src/pages/Strategies/components/StrategyCard/StrategyCard.tsx

```diff
@@ -1,56 +1,53 @@
+import Big from 'big.js';
+import { ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Link } from 'react-router-dom';
 
 import { formatPercentage } from '@/common/utils/utils';
 import { Card, CardProps, CoinIcon, Flex, H1, P, Strong } from '@/component-library';
-import { STRATEGIES, StrategyType } from '@/types/strategies';
-import { PAGES } from '@/utils/constants/links';
 
-import { useGetStrategyInsights } from '../../hooks/use-get-strategy-insights';
+import { StrategyRisk } from '../../types';
 import { StrategyTag } from '../StrategyTag';
 import { StyledEarningCard, StyledEarnSection } from './StrategyCard.style';
 
 type Props = {
-  strategyType: StrategyType;
+  title: ReactNode;
+  description: ReactNode;
+  ticker: string;
+  risk: StrategyRisk;
+  interestRate: Big;
 };
 
 type InheritAttrs = Omit<CardProps, keyof Props>;
 
 type StrategyCardProps = Props & InheritAttrs;
 
-const StrategyCard = ({ strategyType, ...props }: StrategyCardProps): JSX.Element => {
+const StrategyCard = ({ title, description, ticker, risk, interestRate, ...props }: StrategyCardProps): JSX.Element => {
   const { t } = useTranslation();
-  const { interest } = useGetStrategyInsights(strategyType);
-  const { descriptionCard, title, currency, risk, path, tags } = STRATEGIES[strategyType];
 
-  const interestPercentageLable = formatPercentage(interest);
+  const interestPercentageLable = formatPercentage(interestRate.toNumber());
 
   return (
-    <Link to={`${PAGES.STRATEGIES}/${path}`}>
-      <Card {...props} alignItems='center' gap='spacing4'>
-        <Flex alignSelf='flex-start' gap='spacing1'>
-          <StrategyTag risk={risk} />
-          {tags.map((tag) => (
-            <StrategyTag key={tag}>{tag}</StrategyTag>
-          ))}
-        </Flex>
-        <CoinIcon size='xl2' ticker={currency.ticker} />
-        <H1 weight='bold' size='base' align='center' rows={1}>
-          {title}
-        </H1>
-        <StyledEarningCard justifyContent='center'>
-          <StyledEarnSection size='xs' align='center'>
-            Earn up to
-            <Strong size='base'>
-              {interestPercentageLable} {t('apy')}
-            </Strong>
-          </StyledEarnSection>
-        </StyledEarningCard>
-        <P color='tertiary' size='xs' align='center'>
-          {descriptionCard}
-        </P>
-      </Card>
-    </Link>
+    <Card {...props} alignItems='center' gap='spacing4'>
+      <Flex alignSelf='flex-start' gap='spacing1'>
+        <StrategyTag risk={risk} />
+        {risk === StrategyRisk.LOW && <StrategyTag>Passive Income</StrategyTag>}
```

Comment:
NIT: Let's add this to translation file. 

Also, should the passive income tag be always displayed automatically when there is low risk?

Reply:
At least so far ou designs always show the passive income tag for low risk. This can change when there is a different scenario.